live-examples/js-examples/set/set-prototype-values.html

```diff
@@ -3,7 +3,7 @@
 set1.add(42);
 set1.add('forty two');
 
-var iterator1 = set1.values();
+let iterator1 = set1.values();
```

Comment:
Here also const will work. @wbamberg what do you think?

Reply:
Agreed.